packages/visualizations/src/components/Chart/Chart.svelte

```diff
@@ -185,29 +198,47 @@
             {#if options.description}
                 <p id={chartId} class="a11y-invisible-description">{options.description}</p>
             {/if}
-        </div>
+            {#if options?.legend?.custom}
+                <figcaption>
+                    <CategoryLegend {legendOptions} />
+                </figcaption>
+            {/if}
+        </figure>
         {#if options.source}
             <SourceLink source={options.source} />
         {/if}
-    </figure>
+    </div>
 {/if}
 
 <style>
-    figure {
+    .container {
         display: flex;
         flex-direction: column;
-        margin: 0;
+        width: 100%;
     }
-
-    figcaption {
+    .header {
         width: 100%;
         margin: 0;
     }
 
-    .chart-container {
+    figure {
         position: relative;
         width: 100%;
         flex-grow: 1;
+        margin: 0;
+    }
+    .legend--bottom {
+        flex-direction: column;
+    }
+    .legend--right {
+        flex-direction: row;
+    }
```

Comment:
It doesn't seem to work for me, I have a message in the inspect tools saying : `The display: block property prevents flex-direction from having an effect.` and the legend position never changes whatever option I pass. It seems that there is a default `display: block` on the figure html element.